test/integration/image-optimizer/test/util.ts

```diff
@@ -40,11 +40,8 @@ export async function serveSlowImage() {
     res.end(await fs.readFile(join(__dirname, '../app/public/test.png')))
   })
 
-  await new Promise((resolve, reject) => {
-    server.listen(port, (err) => {
-      if (err) return reject(err)
-      resolve()
-    })
```

Comment:
Why was err removed here?

Reply:
To make TS happy. It's non-existent according to TypeScript. The docs does not seem to specify this either: https://nodejs.org/api/net.html#serverlistenoptions-callback